coffeescript/README.md

```diff
@@ -0,0 +1,723 @@
+# CoffeeScript Style Guide
+
+Don’t write new CoffeeScript please. Only use this guide if you’re updating existing CoffeeScript and converting is out of scope.
```

Comment:
Can we bolt this or something? Or maybe get a small bolded blurb that says something like `Note: CoffeeScript is deprecated at Shopify, please do not write any new CoffeeScript files.`